.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
       - master
 
 concurrency:
-  group: ${{ github.head_ref }}
+  group: ${{ github.ref_name }}-${{ github.workflow }}
   cancel-in-progress: true
 
 env:
```

Comment:
I forgot to drop this as well, I believe we have this defined in the [foreman_plugin.yml](https://github.com/theforeman/actions/blob/v0/.github/workflows/foreman_plugin.yml#L41)

Reply:
oh right. want me to clean it up in this PR too?

Reply:
done